src/firekit.ts

```diff
@@ -1331,6 +1331,19 @@ export class RoarFirekit {
     }
   }
 
+  public async getTasksDictionary() {
+    this._verifyAuthentication();
+    const taskDocs = await getDocs(this.dbRefs!.app.tasks);
+
+    // Create a map with document IDs as keys and document data as values
+    const taskMap = taskDocs.docs.reduce((acc, doc) => {
+      acc[doc.id] = doc.data();
+      return acc;
+    }, {} as Record<string, any>);
```

Comment:
```suggestion
    }, {} as Record<string, object>);
```
Maybe object would be the type here? Since task documents return a map with various data types as values.